### PR TITLE
(#2179309) doc: add downstream CONTRIBUTING document

### PR DESCRIPTION
**Pull request by jamacku**
rhel-only

Related: #2179309

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    B -->|NO| D(upstream\nsubmit issue or PR)
+    D --> E{accepted\nand fixed}
+    E -->|YES| C
+    E -->|NO| F(rhel-only patch) --> G
+    C --> G(submit PR)
+```
+
+## Filing issues
+
+When you find an issue with systemd used in CentOS Stream or RHEL, please file an issue by using Bugzilla ticket systemd - [link](https://bugzilla.redhat.com/enter_bug.cgi?classification=Red%20Hat)
```
Comment (dtardon) [inline CONTRIBUTING.md]: Shouldn't this be Jira instead?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Resolves: #1234567
+```
+
+### Issue reference
+
+Each commit has to reference the relevant approved systemd issue (see: [Filling issues section](#filing-issues)). For referencing issues, we use the following keywords:
+
+- **Resolves** for commits that directly resolve issues described in a referenced tracker
+- **Relates** for commits related to the referenced issue, but they don't fix it. Usually, tests and documentation.
```
Comment (dtardon) [inline CONTRIBUTING.md]: s/Relates/Related/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- **Reverts** for commits that reverts previously merged commit
```
Comment (dtardon) [inline CONTRIBUTING.md:52]: This section should also explain the format of the reference, not leave it just for an example.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jamacku) [reply CONTRIBUTING.md]: You are right. It should be possible to use only Jira for new bugs. That is a good point.

File: CONTRIBUTING.md (hunk 3 above)
Comment (jamacku) [reply CONTRIBUTING.md:52]: Added

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    B -->|NO| D(upstream\nsubmit issue or PR)
+    D --> E{accepted\nand fixed}
+    E -->|YES| C
+    E -->|NO| F(rhel-only patch) --> G
+    C --> G(submit PR)
+```
+
+## Filing issues
+
+When you find an issue with systemd used in CentOS Stream or RHEL, please file an issue by using Jira ticket system - [link](https://issues.redhat.com/secure/CreateIssue!default.jspa)
```
Comment (dtardon) [inline CONTRIBUTING.md]: I'd formulate this paragraph as follows:

When you find an issue with systemd used in CentOS Stream or RHEL, please file an issue in [Jira ticket system](https://issues.redhat.com/secure/CreateIssue!default.jspa) (set Project to **RHEL** and Component to **systemd**).

But that's mostly a matter of preference...

Comment (dtardon) [reply CONTRIBUTING.md]: In any case, the link can be set on the words "Jira ticket system".

Comment (jamacku) [reply CONTRIBUTING.md]: But I like your preference :) :+1:

Comment (jamacku) [reply CONTRIBUTING.md]: Done. Thanks for the feedback.

**Review by dtardon — APPROVED** [review]
LGTM

**Pull request merged**